README.md

```diff
@@ -12,14 +12,34 @@ all the heartbleed.
 
 Conduit is **experimental**. Currently, it _only supports HTTP/2_ and is especially
 well-suited for [gRPC][grpc]. Follow our progress towards production-readiness here and on
-[twitter][twitter].
+[Twitter][Twitter].
 
 <!-- TODO add roadmap link -->
 
+## Documentation
+
+View [Conduit docs][conduit-docs] for more a more comprehensive guide to getting started with Conduit.
+
+___
+
+## Getting started with Conduit
+
+### Install the Conduit CLI `curl https://run.conduit.io/install | sh `
+
+### Copy `/Users/username/.conduit/bin/conduit` into your PATH
+
+### Run `conduit install | kubectl apply -f -` to install Conduit
+
+### Check to make sure you're running the latest version _(0.1.0)_ of Conduit: `conduit version`
+
+### Open the local version the Conduit controller with `conduit dashboard`
```

Comment:
Rather than rendering these steps as headers, I might try using a [numbered list](https://help.github.com/articles/basic-writing-and-formatting-syntax/#lists) instead.